### PR TITLE
✨Source Plaid: Migrate Python CDK to No-code CDK

### LINKED ISSUE #28935: plaid

### PR DESCRIPTION
**Pull request by masyagin1998**
## What
* add no-code version of Plaid connector (closes #28935)

## How
* created Plaid no-code version in Airbyte Connector Builder and then put it to new source

## Recommended reading order
* manifest.yaml

## 🚨 User Impact 🚨
* no

## Pre-merge Actions
* review

### CODE REVIEW

**Comment by masyagin1998** [comment]
I think, I can add support of translations stream without any changes to low-code SDK if I use `/transactions/get` instead of `/transactions/sync`, but I'm not sure if it's okay.

Also the bug is still present: records in single response are not filtered by date

**Comment by masyagin1998** [comment]
Now all streams are working and acceptance_test succeeds!
I decided not to remove original `transaction` stream to show its problems.

**Comment by masyagin1998** [comment]
@marcosmarxm @sajarin ready for review!

**Comment by marcosmarxm** [comment]
@masyagin1998 can you share the output from integration tests?

**Comment by masyagin1998** [comment]
@marcosmarxm sure in 4 hours

**Comment by masyagin1998** [comment]
Screenshot from 2023-08-11 23-53-51
Hey @marcosmarxm , done!

**Comment by masyagin1998** [comment]
@marcosmarxm added results to all my other connectors too)

**Comment by masyagin1998** [comment]
Hey @sajarin and @sh4sh !
Do you know how can I fight this error: ```echo "PR is from a fork. Exiting workflow..."``` ?
Many thanks for your reviews and fixes!!!

**Comment by sajarin** [comment]
> Hey @sajarin and @sh4sh ! Do you know how can I fight this error: `echo "PR is from a fork. Exiting workflow..."` ? Many thanks for your reviews and fixes!!!

hey @masyagin1998 due to security reasons with our integration secrets, CI jobs need to ran on a branch with master access, so the error you mentioned will always appear on your PRs since you're not a member of the airbyte organization. We will run tests in a different PR and merge this one, once the tests are passing!

**Comment by masyagin1998** [comment]
@sajarin thank you for answer!
I know the reason: I have to work at Airbyte :D

File: airbyte-integrations/connectors/source-plaid/metadata.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: ed799e2b-2158-4c66-8da4-b40fe63bc72a
-  dockerImageTag: 0.3.2
+  dockerImageTag: 0.4.2
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-plaid/metadata.yaml:5]:
```suggestion
  dockerImageTag: 0.4.0
```

File: airbyte-integrations/connectors/source-plaid/Dockerfile [hunk 2]
```diff
@@ -34,5 +34,5 @@ COPY source_plaid ./source_plaid
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.3.2
+LABEL io.airbyte.version=0.4.2
```
Comment (sh4sh) [inline airbyte-integrations/connectors/source-plaid/Dockerfile:37]:
```suggestion
LABEL io.airbyte.version=0.4.0
```

**Review by sh4sh — APPROVED** [review]

**Comment by sh4sh** [comment]
Thanks for your contribution!

**Pull request merged**